inc/notificationtargetticket.class.php

```diff
@@ -165,6 +165,15 @@ function getDataForObject(CommonDBTM $item, array $options, $simple = false) {
       $data = parent::getDataForObject($item, $options, $simple);
       /*$data['##ticket.description##'] = Html::clean($data['##ticket.description##']);*/
 
+      // Double encode emails stored between '<' and '>' tags
+      // Common case of this is when the ticket was created from a forwarded email
+      // Without double encoding, these emails are interpreted as html and not
+      // rendered in the final mail
+      $regex = "/(&lt;[A-Za-z0-9.!#$%&'*+\-\/=?^_`{|}~]*?@[A-Za-z0-9.!#$%&'*+\-\/=?^_`{|}~]*?&gt;)/";
```

Comment:
We are not validation the email address, so maybe the regex could be simplier.

Reply:
Ok, simplified it to accept any chars expect ";" (= encoded tag end).